### PR TITLE
Minor Nar-Sie/Hell Rising changes

### PR DESCRIPTION
**Pull request by Sprok0**
Summoning Nar-Sie sets the escape shuttle timer to three minutes.
The cult monster portals don't spawn on the centcomm z-level.
All simple animals now properly die when Hell Rising begins.

### CODE REVIEW

**Comment by NigglyWiggly** [comment]
> All simple animals now properly die when Hell Rising begins.

Are they supposed to?
Why?

**Comment by Sprok0** [comment]
I don't know why, but it set stat = DEAD to all animals, which was then undone by their next Life. This just changes it to actually kill them.

**Comment by clusterfack** [comment]
Well what time does it get set to then

**Comment by Sprok0** [comment]
It doesn't set the timer, it just calls the shuttle and disables recalling.

**Comment by clusterfack** [comment]
To be honest I think the timer it set should be reduced to something like 3 minutes.

File: code/game/gamemodes/cult/hell_universe.dm [hunk 1]
```diff
@@ -146,5 +146,5 @@ In short:
 /datum/universal_state/hell/proc/KillMobs()
 	for(var/mob/living/simple_animal/M in mob_list)
```
Comment (ghost) [inline code/game/gamemodes/cult/hell_universe.dm:147]: Wouldnt this kill all cultist constructs as well?

Comment (Sprok0) [reply code/game/gamemodes/cult/hell_universe.dm:147]: It checks if the animal has a client, so it shouldn't kill constructs

**Comment by SkowronX** [comment]
Why not just make hell versions of the pets?

**Comment by coldcola** [comment]
so what would happen with the shuttle when narsie is summoned?

**Comment by clusterfack** [comment]
>Why not just make hell versions of the pets?

Adorable!

**Comment by Sprok0** [comment]
The shuttle is called and the timer is set to three minutes, if the timer is already less than that nothing happens, if the shuttle is at the station or has already left the station, nothing happens. If the shuttle was heading back because it was recalled, it makes a u-turn and heads to the station

If someone can supply sprites for hellpets I could probably do that, but it should probably be in another PR

**Comment by NigglyWiggly** [comment]
> Hell pets

Sign me the fuck up

**Comment by gbasood** [comment]
@NigglyWiggly when you sprite it

**Pull request merged**